dispatcher/dispatcher.go

```diff
@@ -210,14 +215,25 @@ func (d *IotxDispatcher) blockHandler() {
 	}
 }
 
+func (d *IotxDispatcher) checkSyncPermission(peerID string) (bool) {
+	now := time.Now()
+	last, ok := d.peerLastSync[peerID]
+	if ok && last.Add(d.syncInterval).After(now) {
+		return false
+	}
+
+	d.peerLastSync[peerID] = now
+	return true
+}
+
 // syncHandler handles incoming block sync requests
 func (d *IotxDispatcher) syncHandler() {
 	for {
 		select {
 		case m := <-d.syncChan:
-			d.handleBlockSyncMsg(m)
-			// TODO: better strategy of pacing
-			time.Sleep(5 * time.Second)
+			if (d.checkSyncPermission(m.peer.ID.Pretty())) {
```

Comment:
if checkSyncPermission == false
the request is simply discarded
i don't think this is a good strategy to serve the sync request?
for example, request [0, 16], [17, 32], [33, 48], received by A, B, A
A first serves [0, 16], then discard [33, 48] due to syncInterval has not passed

Reply:
we definitely need a better way other than `time.Sleep(5 * time.Second)`, but IMO maybe not simply discarding the request, can we think of a better solution?